pkg/stis/calstis/cs1/dodqi.c

```diff
@@ -652,6 +653,8 @@ int doppmin, doppmax  i: offsets for Doppler shift
 	    xlength = 2;
 	    ylength = tabrow->length * 2;
 	}
+	else
+	    assert(0);
```

Comment:
I agree to all (as stated by opening caveat) that this is not 'the' fix but simply a temporary prevention of buggy results.

@sosey yep, I should have been more explicit and expanded my example, and also remembered that C has no exception handling and thus there's no way for the try-catch block that I assumed existed within all of our main() funcs to actually exist.